fulp_modules/features/species/beefmen/mob/beefman.dm

```diff
@@ -247,6 +265,37 @@
 		user.put_in_hands(dropped_meat)
 	return TRUE
 
+/datum/species/beefman/spec_attacked_by(var/obj/item/organ/tongue/beefman/meaty_tongue, mob/living/user, obj/item/bodypart/affecting, mob/living/carbon/human/beefboy)
```

Comment:
why arent you using the proc below

Reply:
I tried to at first but it turned into a mess coz organs and LIMBS work very differently. I can still try to shove both of them into the same proc if you want

Reply:
you have to
this code is unacceptable otherwise

Reply:
how is it NOW

Reply:
i tested all imaginable cases it works